### PR TITLE
Connect with API

### PR DESCRIPTION
**Pull request by asifaminisonline**
### In this step of the project :

redux-thunk is added to the project
API calls are created using fetch to GET a list of books, POST a new book and DELETE a book by id.
Apply JavaScript Best Practices.
Use Git Flow.
No Linter errors.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -3,27 +3,32 @@ const ADD = 'add';
...
+        ...state,
+        {
+          ...action.payload,
+        },
+      ];
     case DEL:
       return [...state.filter((book) => book.id !== action.payLoad.id)];
-    default: return state;
+    default:
+      return state;
```
Comment (mckent05) [inline src/redux/books/books.js]: - Hello @asifaminisonline, you have not properly connected your app to the API. There is no `createAsyncThunk` used in the project for creating thunk action creators 🙏
- Please ensure that all the tasks in the [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/connec_to_api.md) have been fully achieved. Please fix this 🙏

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @asifaminisonline, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asifaminisonline** [comment]
Add changes

**Review by DammyShittu — APPROVED** [review]
Hi @asifaminisonline,

## APPROVED 🟢

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**